v2/fleet/datadog-control.service

```diff
@@ -0,0 +1,27 @@
+# https://raw.githubusercontent.com/DataDog/docker-dd-agent/master/dd-agent.service
+# make sure the etcd /ddapikey is set!
+[Unit]
```

Comment:
We also need `ddmesoshost` set in `etcd`. The value needs to be the control ELB.